packages/cli/src/lib/collect/command-object.spec.ts

```diff
@@ -1,56 +1,43 @@
-import { Report } from '@quality-metrics/models';
-import { dummyConfig } from '@quality-metrics/models/testing';
-import { CollectOptions } from '@quality-metrics/core';
-import { readFileSync } from 'node:fs';
-import { join } from 'node:path';
-import { yargsCli } from '../cli';
-import { logErrorBeforeThrow } from '../implementation/utils';
+import { yargsCli } from '../yargs-cli';
 import { middlewares } from '../middlewares';
-import { yargsGlobalOptionsDefinition } from '../implementation/global-options';
-import { yargsCollectCommandObject } from './command-object';
-import { getDirname } from '../implementation/helper.mock';
+import { options } from '../options';
+import { vi } from 'vitest';
+import { objectToCliArgs } from '@code-pushup/utils';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+import { yargsUploadCommandObject } from '../upload/command-object';
 
-const command = {
-  ...yargsCollectCommandObject(),
-  handler: logErrorBeforeThrow(yargsCollectCommandObject().handler),
-};
-
-const outputPath = 'tmp';
-const reportPath = (format: 'json' | 'md' = 'json') =>
-  join(outputPath, 'report.' + format);
-
-const config = dummyConfig();
+const baseArgs = [
+  ...objectToCliArgs({
+    verbose: true,
+    configPath: join(fileURLToPath(dirname(import.meta.url)), 'config.mock.ts'),
+  }),
+];
+const cli = (args: string[]) =>
+  yargsCli(['collect', ...args], {
+    options,
+    middlewares,
+    commands: [yargsUploadCommandObject()],
```

Comment:
It's the `collect` command test, so this shouldn't be using the `upload` command.